### PR TITLE
MAINT: Bootstrap update

### LINKED ISSUE #132: Bootstrap version update
## The problem

Bootstrap version is not consistent across the base.html.

Currently using:
twitter-bootstrap/4.0.0-beta.3
bootstrap/3.3.7

Need to move to 5.0 as decided in afids meeting on May 26. Data toggle will need renaming as a result.

### PR DESCRIPTION
**Pull request by kaitj**
## Proposed changes

Resolves #132  - updates bootstrap to v5.0.1. Navbar and plotting tabs have been updated to make use of new/renamed classes and variables. Other changes include:
* Updated CSS to use only what is needed
* Updated `col-*` classes in `index.html` to use `col-*` instead of `col-sm-*`, which was originally meant for different screen sizes (e.g. mobile vs desktop)
* Centered the video within the `col-5` on index.html
* Font size (which was not previously set), now set to 2.5% of viewport width for navigation headings
* Icon sizes in footer updates
* Removed `figcaption` class from CSS, which was never used AFAIK

I have also run `prettier` on the files I have touched, so some of the formatting may have changed from before.

## Types of changes

What types of changes does your code introduce? Put an `x` in the boxes that apply

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionalitiy)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (if none of the other choices apply)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you are unsure about any of the choices, don't hesitate to ask!

- [x] Changes have been tested to ensure that fix is effective or that a feature works.
- [x] Changes pass the unit tests
- [x] Code has been run through [`black`](https://github.com/psf/black) with the `-l 79` flag.
- [x] I have included necessary documentation or comments (as necessary)
- [ ] Any dependent changes have been merged and published

## Notes
All PRs will undergo the unit testing before being reviewed. You may be requested to explain or make additional changes before the PR is accepted.

_PR template was adopted from [appium](https://github.com/appium/appium/blob/master/.github/PULL_REQUEST_TEMPLATE.md)_

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -1,54 +1,107 @@
...
+    />
+    <link
+      rel="stylesheet"
+      href="{{ url_for('static', filename='css/afids_custom.css') }}"
+      type="text/css"
+    />
+  </head>
+
+  <body class="d-flex flex-column h-100">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.js"></script>
```
Comment (tkkuehn) [inline templates/base.html:47]: Do we need jquery anymore? Not sure what we're using it for...

File: templates/base.html [hunk 2]
```diff
@@ -1,54 +1,107 @@
...
+    <script
```
Comment (tkkuehn) [inline templates/base.html:48]: This should go at the end of the body per [the docs](https://getbootstrap.com/docs/5.0/getting-started/introduction/#js).

File: static/css/afids_custom.css [hunk 3]
```diff
@@ -1,44 +1,54 @@
...
 }
 
-footer {
-    background-color: #00ff01;
-    color: #000;
+/* Nav */
+header .nav-link {
+  font-family: "Bebas Neue", serif;
+  font-size: 2vw;
+  text-transform: uppercase;
```
Comment (tkkuehn) [inline static/css/afids_custom.css:15]: Could set `.text-uppercase` on the links instead.

File: static/css/afids_custom.css [hunk 4]
```diff
@@ -1,44 +1,54 @@
...
+  color: #fff;
 }
 
-footer {
-    background-color: #00ff01;
-    color: #000;
+/* Nav */
+header .nav-link {
+  font-family: "Bebas Neue", serif;
+  font-size: 2vw;
```
Comment (tkkuehn) [inline static/css/afids_custom.css:14]: I'd prefer to set this with a [bootstrap class](https://getbootstrap.com/docs/5.0/utilities/text/#font-size) or by making it a header.

File: static/css/afids_custom.css [hunk 5]
```diff
@@ -1,44 +1,54 @@
...
+  text-transform: uppercase;
+  margin-left: 1vw;
+  margin-right: 1vw;
```
Comment (tkkuehn) [inline static/css/afids_custom.css:17]: Could set these with [`.mx-*`](https://getbootstrap.com/docs/5.0/utilities/spacing/#margin-and-padding) bootstrap classes.

File: static/css/afids_custom.css [hunk 6]
```diff
@@ -1,44 +1,54 @@
...
+  background-color: #00ff01;
+  color: #000;
 }
 
-.nav a:hover {
-    text-decoration: none;
-    color: #00ff01;
+footer .nav-link {
+  color: #000;
+  font-size: 1vw;
```
Comment (tkkuehn) [inline static/css/afids_custom.css:44]: Could set this with bootstrap instead.

**Review by tkkuehn — CHANGES_REQUESTED** [review]
Looking good, mostly small changes to think about. Thanks for taking care of this!

File: static/css/afids_custom.css (hunk 5 above)
Comment (kaitj) [reply static/css/afids_custom.css:17]: I forgot those exists - agree with using the bootstrap class if possible

File: static/css/afids_custom.css (hunk 4 above)
Comment (kaitj) [reply static/css/afids_custom.css:14]: Set font-size static

File: static/css/afids_custom.css (hunk 6 above)
Comment (kaitj) [reply static/css/afids_custom.css:44]: Font-size (see above)

File: templates/base.html (hunk 1 above)
Comment (kaitj) [reply templates/base.html:47]: Removed in 2b1da4b

File: templates/base.html (hunk 2 above)
Comment (kaitj) [reply templates/base.html]: Moved in 2b1da4b

File: static/css/afids_custom.css (hunk 3 above)
Comment (kaitj) [reply static/css/afids_custom.css]: Added `text-uppercase` class to the `nav` and removed from css in 8d73355

File: static/css/afids_custom.css (hunk 6 above)
Comment (kaitj) [reply static/css/afids_custom.css]: Updated in 61627ea

File: static/css/afids_custom.css (hunk 5 above)
Comment (kaitj) [reply static/css/afids_custom.css]: Updated in 114ab99

File: static/css/afids_custom.css (hunk 4 above)
Comment (kaitj) [reply static/css/afids_custom.css]: Updated in 14ab99

**Comment by kaitj** [comment]
Everything seemed to be working without jquery so I am at a loss at what it was used for. Anyways, it is now removed and ready for another review.

**Review by tkkuehn — APPROVED** [review]
Looks good to me!

**Pull request merged**